CONTRIBUTING.md

```diff
@@ -490,3 +491,13 @@ We are running a Discord server. You can go here to talk, discuss, and more with
     <li>Create a <a href="https://github.com/devicons/devicon/releases/new">new release</a> using the format "<b>Release v<i>MAJOR</i>.<i>MINOR</i>.<i>PATCH</i></b>" as tag and release title. Use the earlier created description as description of the release.</li>
     <li>Publishing the release will trigger the <a href="/.github/workflows/npm_publish.yml">npm_publish.yml</a> workflow which will execute a <code>npm publish</code> leading to a updated <a href="https://www.npmjs.com/package/devicon">npm package</a> (v<i>MAJOR</i>.<i>MINOR</i>.<i>PATCH</i>).</li>
 </ol>
+
+<h2 id='resources'>Recommended resources and tools</h2>
+
+| Tool Name          | Link                                               | Description & Usage                                |
+| :----------------- | :------------------------------------------------- | :------------------------------------------------- |
+| Inkscape           | https://inkscape.org/                              | Desktop application for editing and Making SVG's   |
```

Comment:
```suggestion
| Inkscape           | https://inkscape.org/                              | Desktop application for editing and making SVG's   |
```

Reply:
already merged :S

Reply:
Want to make another PR to fix this? :)

Reply:
Yes, but I don't think I have time until next year. Feel free to make a PR if you want the change before that :)

Reply:
It's alright. I'm updating the docs and converting it into the GitHub wikis anyway. I'll watch out for this.